tests/test_reified_generics/test_reified_generic.py

```diff
@@ -79,9 +78,7 @@ class SubASpecified(A[int]):
 
 
 def test_none_type():
-    # TODO: is this mypy error correct?
-    #  https://github.com/KotlinIsland/basedtyping/issues/74
-    assert Reified[None, None].__reified_generics__ == (NoneType, NoneType)  # type:ignore[comparison-overlap]
+    assert Reified[None, None].__reified_generics__ == (NoneType, NoneType)
```

Comment:
can that issue be closed (i think we deleted the TODO checker github action so it wont be closed automatically)